### ISSUE TITLE
Always show selected values in `Combobox`

### ISSUE THREAD

**Issue by mimarz** [digdir/designsystemet#1519]
We should always show selected options even when people have search for as part of being clear and concise for our users what is selected in the `Combobox`. 

This is especially important for screen-readers where; 
- It is not necessarily intuitiv that you need to tab back to chips to get selected options and deselect options
- Screen readers should be able to read out how many selected options in the list, such as 2 of 8 etc. 

```[tasklist]
### Tasks
- [ ] https://github.com/digdir/designsystemet/issues/1564
- [ ] https://github.com/digdir/designsystemet/issues/1565
```

**Comment by mimarz** [comment]
We should maybe update the examples in Figma to reflect this? @Febakke
